packages/beachball/src/__tests__/bump/updateRelatedChangeType.test.ts

```diff
@@ -102,6 +102,81 @@ describe('updateRelatedChangeType', () => {
     expect(bumpInfo.packageChangeTypes['unrelated']).toBeUndefined();
   });
 
+  it('should bump all packages in a group together as patch', () => {
+    const bumpInfo = _.merge(_.cloneDeep(bumpInfoFixture), {
+      dependentChangeTypes: {
+        foo: 'patch',
+      },
+      packageInfos: {
+        foo: {
+          group: 'grp',
+        },
+        bar: {
+          group: 'grp',
+        },
+        unrelated: {},
+      },
+      packageGroups: { grp: { packageNames: ['foo', 'bar'] } },
+    });
+
+    updateRelatedChangeType('foo', 'patch', bumpInfo, true);
+
+    expect(bumpInfo.packageChangeTypes['foo']).toBe('patch');
+    expect(bumpInfo.packageChangeTypes['bar']).toBe('patch');
+    expect(bumpInfo.packageChangeTypes['unrelated']).toBeUndefined();
+  });
+
+  it('should bump all packages in a group together as none', () => {
+    const bumpInfo = _.merge(_.cloneDeep(bumpInfoFixture), {
+      dependentChangeTypes: {
+        foo: 'patch',
+      },
+      packageInfos: {
+        foo: {
+          group: 'grp',
+        },
+        bar: {
+          group: 'grp',
+        },
+        unrelated: {},
+      },
+      packageGroups: { grp: { packageNames: ['foo', 'bar'] } },
+    });
+
+    updateRelatedChangeType('foo', 'none', bumpInfo, true);
+
+    expect(bumpInfo.packageChangeTypes['foo']).toBe('none');
+    expect(bumpInfo.packageChangeTypes['bar']).toBe('none');
+    expect(bumpInfo.packageChangeTypes['unrelated']).toBeUndefined();
+  });
+
+  it('should bump all packages in a group together as none with dependents', () => {
+    const bumpInfo = _.merge(_.cloneDeep(bumpInfoFixture), {
+      dependents: {
+        foo: ['bar'],
+      },
+      dependentChangeTypes: {
+        foo: 'none',
```

Comment:
if pkgs inside a group take dependency on each other and this is `patch` (which is what we set as default when all change files), we still get `patch` bump for all packages inside the group despite the only change inside the group is `none` type